eng/Tools.props

```diff
@@ -5,8 +5,8 @@
     <PackageReference Include="Microsoft.DotNet.VersionTools.Tasks" Version="$(MicrosoftDotNetVersionToolsTasksVersion)" />
 
     <!-- SourceLink -->
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
-    <PackageReference Include="Microsoft.SourceLink.AzureRepos.Git" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.SourceLink.AzureRepos.Git" Version="$(MicrosoftSourceLinkAzureReposGitVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
```

Comment:
Why do these PackageReferences even exist? I don't think we need them. Maybe something to follow-up post-merge.

Reply:
Good question? They exist in a few places, runtime, windowsdesktop, deployment-tools. The comment in deployment-tools may be illuminating? https://github.com/dotnet/deployment-tools/pull/227/files#diff-008293a45f05d88c0dba977ec62198b6f0f011e02e44619fa09d9f58e82cded7L10

Reply:
@ericstj might know about pkgproj's use case of sourcelink but at least windowsdesktop shouldn't need it as it doesn't have pkgprojs anymore.

Reply:
That was in support of a source-only package type.  Here's the original issue: https://github.com/dotnet/runtime/issues/37921

My guess is that these other repos copy pasted some changes without understanding them.